atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/NoOpInternalLockWatchManager.java

```diff
@@ -19,16 +19,20 @@
 import java.util.Optional;
 import java.util.Set;
 
+import com.palantir.lock.client.CommitUpdateGetter;
+import com.palantir.lock.v2.LockToken;
+import com.palantir.lock.watch.CommitUpdate;
 import com.palantir.lock.watch.IdentifiedVersion;
 import com.palantir.lock.watch.LockWatchReferences;
 import com.palantir.lock.watch.NoOpLockWatchEventCache;
 import com.palantir.lock.watch.TransactionsLockWatchEvents;
 
 public final class NoOpInternalLockWatchManager implements InternalLockWatchManager {
-    public static final InternalLockWatchManager INSTANCE = new NoOpInternalLockWatchManager();
 
-    private NoOpInternalLockWatchManager() {
-        // ...
+    private final CommitUpdateGetter commitUpdateGetter;
```

Comment:
this seems wrong - surely it should just return a random response?

Reply:
ah, I see, you construct it elsewhere. I think you should just explicitly return empty here - it shouldn't need an actual commit update getter